spec/spec_helper.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "simplecov"
+
+unless ENV["NO_COVERAGE"]
+  SimpleCov.start do
+    add_filter %r(^/spec/)
+    enable_coverage :branch
+    enable_coverage_for_eval
+    minimum_coverage_by_file line: 95, branch: 95
+  end
+end
+
+Bundler.require :tools
+
+require "dry/operation"
+
+SPEC_ROOT = Pathname(__dir__).realpath.freeze
+
+RSpec.configure do |config|
+  config.color = true
+  config.disable_monkey_patching!
+  config.example_status_persistence_file_path = "./tmp/rspec-examples.txt"
+  config.filter_run_when_matching :focus
+  config.formatter = ENV.fetch("CI", false) == "true" ? :progress : :documentation
```

Comment:
Does this mean it uses the `documentation` formatter when running tests locally? If so, I think I'd prefer we'd just stick with the `progress` format there too. I'd like to keep the output concise for something that is run so often locally. And there's always `rspec spec --format doc` for when the opposite is preferred :)

Reply:
Amended. The setting is progress only now.

I used to configure RSpec as progress only but over the years I've realized that I value the documentation always being there because it's a great refresher/reminder of how the implementation works. I also like all of the output for debugging purposes. It's like a friendly coding buddy that I miss when they aren't there.